onnxruntime/core/session/inference_session.cc

```diff
@@ -1997,7 +1997,8 @@ Status InferenceSession::Run(const RunOptions& run_options,
 
     // info all execution providers InferenceSession:Run ended
     for (auto* xp : exec_providers_to_stop) {
-      auto status = xp->OnRunEnd(run_options.synchronize_execution_providers);
+      bool synchronize_execution_providers = !run_options.config_options.GetConfigEntry(kOrtRunOptionsConfigDisableSynchronization).has_value();
```

Comment:
I would prefer that we not deal with double negatives. It makes code difficult to reason with.

Reply:
Changed by assuming a default value of `"1"`